examples/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "examples"
+version = "0.0.0"
+publish = false
+edition = "2018"
+
+[dev-dependencies]
+scipio = { version = "0.1.0", path = "../scipio" }
+concurrent-queue = "1.1.2"
+futures-lite = "0.1.9"
+libc = "0.2.73"
+socket2 = { version = "0.3.12", features = ["pair", "unix"] }
+iou = { git = "https://github.com/glommer/iou", tag = "poll-ring-works" }
+uring-sys = { git = "https://github.com/glommer/uring-sys", tag= "poll-ring-works" }
+nix = "0.16.0"
+aligned_alloc = "0.1"
+bitmaps = "2.1.0"
+typenum = "1.12"
+scoped-tls = "1.0.0"
+futures = "0.3.5"
+rlimit = "0.3.0"
```

Comment:
Aren't the dependencies transitive? importing `scipio` should import the rest, no?

Reply:
@HippoBaro Yeah, cleaned up them. Had to import `futures`, though...